samples/AnomalyDetector/Generated/AnomalyDetectorClient.cs

```diff
@@ -894,7 +894,7 @@ public virtual AsyncPageable<AnomalyDetectionModel> GetMultivariateModelValuesAs
             RequestContext context = cancellationToken.CanBeCanceled ? new RequestContext { CancellationToken = cancellationToken } : null;
             HttpMessage FirstPageRequest(int? pageSizeHint) => CreateGetMultivariateModelsRequest(skip, maxCount, context);
             HttpMessage NextPageRequest(int? pageSizeHint, string nextLink) => CreateGetMultivariateModelsNextPageRequest(nextLink, skip, maxCount, context);
-            return PageableHelpers.CreateAsyncPageable(FirstPageRequest, NextPageRequest, AnomalyDetectionModel.DeserializeAnomalyDetectionModel, ClientDiagnostics, _pipeline, "AnomalyDetectorClient.GetMultivariateModels", "models", "nextLink", context);
+            return PageableHelpers.CreateAsyncPageable(FirstPageRequest, NextPageRequest, AnomalyDetectionModel.DeserializeAnomalyDetectionModel, ClientDiagnostics, _pipeline, "AnomalyDetectorClient.GetMultivariateModelValues", "models", "nextLink", context);
```

Comment:
These are the only changes in generated code, and they are correct, cause method name is `GetMultivariateModelValuesAsync`, not `GetMultivariateModelsAsync`.

Reply:
This is actually an interesting question.  Is telemetry intended to track client side or service side?

If its service side then `GetMultivariateModels` and `GetMultivariateModelValues` should all share the same scope similar to how we don't have 2 scopes to differentiate between sync and async since they both end up calling the same service operation.

@tg-msft do you have an opinion here?

Reply:
My intuition is we want the scope name to match the method name, modulo the `Async` suffix since that's usually a shared or at least identical code path.  If we have different names between protocol and convenience methods, I think that would imply different scope names.  +@lmolkova to give you a definitive answer though.

Reply:
your intuition is absolutely correct @tg-msft, the public-surface API telemetry represents client-side  (how long logical call took in their app and how it ended). Having method name in the Activity/span name helps users match telemetry to their code.  This Async suffix is not that helpful and I believe we decided not to populate it. 

So this change looks great to me: scope matches method name as per guidelines

Reply:
@lmolkova  thanks for the clarification.